packager/app/test/packager_test.py

```diff
@@ -1538,6 +1538,13 @@ def testEc3AndHlsSingleSegmentMp4Encrypted(self):
             ['audio', 'video'], hls=True, test_files=['bear-640x360-ec3.mp4']),
         self._GetFlags(encryption=True, output_hls=True))
     self._CheckTestResults('ec3-and-hls-single-segment-mp4-encrypted')
+  
+  def testHlsByterange(self):
```

Comment:
I'd prefer calling it: testHlsSingleSegmentTs, as "hls-byte-range" could be for mp4 or ts.